sagemaker-python-sdk/mxnet_gluon_mnist/mnist_with_gluon_local_mode.ipynb

```diff
@@ -0,0 +1,259 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Local MNIST Training with MXNet and Gluon\n",
+    "\n",
+    "### Pre-requisites\n",
+    "\n",
+    "This notebook shows how to use the SageMaker Python SDK to run your code in a local container before deploying to SageMaker's managed training or hosting environments.  This can speed up iterative testing and debugging while using the same familiar Python SDK interface.  Just change your estimator's `train_instance_type` to `local` (or `local_gpu` if you're using an ml.p2 or ml.p3 notebook instance).\n",
+    "\n",
+    "In order to use this feature you'll need to install docker-compose (and nvidia-docker if training with a GPU).\n",
```

Comment:
Let's add one more sentence explicitly stating that the setup.sh cell does this for you.